js/widgets/page.js

```diff
@@ -46,6 +46,7 @@ $.widget( "mobile.page", {
 
 		// Deprecated in 1.4 remove in 1.5
 		contentTheme: null,
+		enhanceWithin: true,
```

Comment:
how about just `enhance` lets keep this simple. This is also the new function name and it being close to `enhanced` is fine they essentially control the same thing.